lib/qless/server.rb

```diff
@@ -127,27 +127,23 @@ def failed
         client.jobs.failed
       end
 
-      def failed_jobs_by_type(tagged: nil, include_not_tagged: true)
-        jobs = client.jobs.failed.map { |k, v| [k, client.jobs.failed(k)["jobs"]] }.to_h
+      def failed_jobs_by_type(include_tag: nil, exclude_tag: nil)
+        jobs = client.jobs.failed.map { |k, _v| [k, client.jobs.failed(k)['jobs']] }.to_h
 
-        tagged_jobs = if tagged
-          jobs.
-            transform_values { |v| v.select { |job| res = job.tags.grep(tagged); res && res.count > 0 } }.
-            select { |k, v| v.length > 0 }
-        else
-          jobs
+        find_tagged = jobs.transform_values do |v|
```

Comment:
Looks like my last comment didn't go through. I think it would be cleaner to do something like:

```
tag_filter = lambda { |j| !j.tags.grep(include_tag).empty? && j.tags.grep(exclude_tag).empty? }
{ 
  tagged: jobs.transform_values { |v| v.select(&tag_filter) }.reject { |_k, v| v.empty? },
  not_tagged: jobs.transform_values { |v| v.reject(&tag_filter) }.reject { |_k, v| v.empty? },
}